plugins/inputs/snmp/README.md

```diff
@@ -22,8 +22,13 @@ information.
 ```toml
 [[inputs.snmp]]
   ## Agent addresses to retrieve values from.
+  ##   format:  agents = ["<scheme://><hostname>:<port>"]
+  ##   scheme:  optional, either udp, udp4, udp6, tcp, tcp4, tcp6.  
+  ##            default is udp
+  ##   port:    optional
   ##   example: agents = ["udp://127.0.0.1:161"]
   ##            agents = ["tcp://127.0.0.1:161"]
+  ##            agents = ["udpv4://v4only-snmp-agent"]
```

Comment:
Is there an extra "v" in the scheme in this example?
```suggestion
  ##            agents = ["udp4://v4only-snmp-agent"]
```

Reply:
Wow, good catch!